tmt/steps/provision/mrack.py

```diff
@@ -512,6 +512,28 @@ def _transform_zcrypt_mode(
         children=[MrackHWKeyValue('ZCRYPT_MODE', beaker_operator, actual_value)])
 
 
+def _transform_iommu_is_supported(
+        constraint: tmt.hardware.FlagConstraint,
+        logger: tmt.log.Logger) -> MrackBaseHWElement:
+
+    test = (constraint.operator, constraint.value)
+
+    if test in [(tmt.hardware.Operator.EQ, True), (tmt.hardware.Operator.NEQ, False)]:
+        return MrackHWKeyValue("VIRT_IOMMU", '==', '1')
```

Comment:
For `VIRT_IOMMU`, double quotation marks are used. But for `==` and `1`, single quotation marks are used. Please make them consistent.